rplugin/python3/deoplete/deoplete.py

```diff
@@ -53,6 +55,13 @@ def __init__(self, vim):
         self.__profile_start = 0
 
     def completion_begin(self, context):
+        if not logger.init:
+            log_file = None
+            if self.__vim.vars['deoplete#enable_debug']:
+                self.debug_enabled = True
+                log_file = self.__vim.vars['deoplete#debug_log']
+            logger.setup(self.__vim, log_file)
```

Comment:
@tweekmonster 
> If self.__vim.vars['deoplete#enable_debug'] is checked each time, that would cause a slowdown (I think). Something needs to know that enable_debug was already checked.

Yes, maybe will check each time.
Alternatively, Is there problems in `int` or `None` check for `self.debug_enabled = ***`?
such as 
https://github.com/Shougo/deoplete.nvim/blob/master/rplugin/python3/deoplete/deoplete.py#L247